third_party/2.7/dateutil/parser.pyi

```diff
@@ -0,0 +1,37 @@
+from typing import List, Tuple, Optional, Callable, Union, IO, Any
+from datetime import datetime
+
+__all__ = ...  # type: List[str]
+
+
+class parserinfo(object):
+    JUMP = ... # type: List[str]
```

Comment:
Because `parser.py` imports `unicode_literals` from `__future_`, `JUMP`, `WEEKDAYS`, `MONTHS`, `HMS`, `AMPM`, `UTCZONE`, `PERTAIN` and `TZOFFSET` are all `list`/`dict`/`tuple`s of `unicode`, not `str`.